app/src/main/java/com/asfoundation/wallet/verification/code/VerificationCodePresenter.kt

```diff
@@ -55,23 +60,31 @@ class VerificationCodePresenter(private val view: VerificationCodeView,
             }
             .subscribe({}, {
               logger.log(TAG, it)
-              view.showGenericError()
+              navigator.navigateToError(VerificationCodeResult.ErrorType.OTHER, data.amount,
+                  data.symbol)
             })
     )
   }
 
   private fun handleAnotherCardClicks() {
     disposable.add(
         view.getChangeCardClicks()
-            .doOnNext { navigator.navigateToInitialWalletVerification() }
+            .observeOn(viewScheduler)
+            .doOnNext {
+              analytics.sendConfirmEvent("try_with_another_card")
```

Comment:
Shouldn't these string be as const in the Analytics class?

Reply:
We don't do this in other cases (you can check any other analytics, we do the same) and I think it's correct. I think it's correct because:
1) The name of these actions is directly tied with their english UI label (according to Ana) and so if you change its name, you should also change here. I think it is easier and intuitive to change here instead of changing in the analytics class.
2) These are unique actions in a screen that are unlikely to be reused, so there is no gain by trying to extract them to variables.

However let me know if you any counter-argument to this since I'm not 100% sure either.